src/utils/Functions/functions.js

```diff
@@ -29,7 +33,198 @@ function throttle(func, limit) {
   };
 }
 
+// GET USER DETAILS
+export const getUserDetails = async (user_id) => {
+  try {
+    // Step 1: Validate the input
+    if (!user_id) {
+      return { message: "User ID is required", status: 400 };
+    }
+
+    // Step 2: Get the user document from Firestore
+    const userDocRef = doc(db, "Users", user_id); // Get reference to the user document
+    const userSnapshot = await getDoc(userDocRef); // Fetch the document snapshot
+
+    // Step 3: Check if the user exists
+    if (!userSnapshot.exists()) {
+      return { message: "User not found", status: 404 };
+    }
+
+    // Step 4: Extract user data
+    const userData = userSnapshot.data();
+    const {
+      id,
+      email,
+      firstName,
+      lastName,
+      linkedin,
+      matchedWith,
+      profile,
+      role,
+      skills,
+    } = userData; // Extract specific fields
```

Comment:
Remove all comments.

Also you don't need the Steps #. I'd argue you don't even need the names for any of these functions